### ISSUE TITLE
Inconsistencies in CLIP features lead to noticeable differences in performance.

### ISSUE THREAD

**Issue by wuyinglong123** [nwpu-zxr/VadCLIP#26]
Dear author，
it is a nice job, but I have some question
 I try to use "ViT-B/32" to exact the feature of the video with 16 frames and 10 crops, but the AUC of UCF-Crime is only 81,and I find the Similarity of vision feature and text feature exacted by CLIP you provided is not high 。and could you provide the model or tell us the name of CLIP -based model?

**Comment by scp147369** [comment]
[We use standard CLIP(ViT-B/16) to extract 10crop features of the videos.](https://github.com/nwpu-zxr/VadCLIP/issues/1#issuecomment-1873627266)

from author reply

**Comment by 12sang3** [comment]
@wuyinglong123 Hello, I would like to ask how many epochs it took you to achieve an AUC of 81. My reproduced AUC is only around 50. I look forward to your response.

**Comment by wuyinglong123** [comment]
几十个epochs，but at last I find his features exacted by CLIPB16 is different from my features exacted by CLIPB16.



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 23, 2024 14:24 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nwpu-zxr/VadCLIP] Inconsistencies in CLIP features lead tonoticeable differences in performance. (Issue #26)




 
@wuyinglong123 Hello, I would like to ask how many epochs it took you to achieve an AUC of 81. My reproduced AUC is only around 50. I look forward to your response.

**Comment by 12sang3** [comment]
Did you use the features from the Baidu Netdisk? If so, did you achieve the result using your own features or the provided ones?

**Comment by wuyinglong123** [comment]
Yes.I used the features from Baidu, the auc is about 87.10. But lastly I used my features,the auc is only 80+



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 23, 2024 14:38 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nwpu-zxr/VadCLIP] Inconsistencies in CLIP features lead tonoticeable differences in performance. (Issue #26)




 
Did you use the features from the Baidu Netdisk? If so, did you achieve the result using your own features or the provided ones?

**Comment by 12sang3** [comment]
Alright, may I ask if you made any modifications during the reproduction process? I generated the ucf_CLIP_rgbtest.csv randomly from the dataset, but the training AUC value has consistently been around 50. I'm not sure if this is the issue.

**Comment by wuyinglong123** [comment]
I have no process



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 23, 2024 14:46 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nwpu-zxr/VadCLIP] Inconsistencies in CLIP features lead tonoticeable differences in performance. (Issue #26)




 
Alright, may I ask if you made any modifications during the reproduction process? I generated the ucf_CLIP_rgbtest.csv randomly from the dataset, but the training AUC value has consistently been around 50. I'm not sure if this is the issue.

**Comment by 12sang3** [comment]
“I have no process” does it mean no changes were made?

**Comment by wuyinglong123** [comment]
yes





---Original---
From: ***@***.***&gt;
Date: Mon, Dec 23, 2024 14:51 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nwpu-zxr/VadCLIP] Inconsistencies in CLIP features lead tonoticeable differences in performance. (Issue #26)




 
“I have no process” does it mean no changes were made?

**Comment by 12sang3** [comment]
Thank you very much. I will continue to give it a try.
